src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl

```diff
@@ -0,0 +1,59 @@
+using NCDatasets
+using Interpolations
+
+function get_OGW_info(Y, orographic_info_rll)
+    FT = Spaces.undertype(axes(Y.c))
+
+    lon, lat, topo_ll = get_topo_ll(orographic_info_rll)
+
+    varnames = ["hmax", "hmin", "t11", "t12", "t21", "t22"]
+    topo_cg = FieldFromNamedTuple(
+        axes(Fields.level(Y.c.ρ, 1)),
+        (;
+            t11 = FT(0),
+            t12 = FT(0),
+            t21 = FT(0),
+            t22 = FT(0),
+            hmin = FT(0),
+            hmax = FT(0),
+        ),
+    )
+
+    cg_lat = Fields.level(Fields.coordinate_field(Y.c).lat, 1)
+    cg_lon = Fields.level(Fields.coordinate_field(Y.c).long, 1)
+
+    for varname in varnames
+        li_obj = linear_interpolation(
+            (lon, lat),
+            getproperty(topo_ll, Symbol(varname)),
```

Comment:
```suggestion
            getproperty(topo_ll, varname),
```

Reply:
varname here is string, I tried a simple test in REPL below
```
julia> a = (;b =1)
(b = 1,)

julia> getproperty(a, "b")
ERROR: MethodError: no method matching getproperty(::NamedTuple{(:b,), Tuple{Int64}}, ::String)
Closest candidates are:
  getproperty(::Any, ::Symbol) at Base.jl:38
  getproperty(::Any, ::Symbol, ::Symbol) at Base.jl:50
Stacktrace:
 [1] top-level scope
   @ REPL[2]:1

julia> getproperty(a, Symbol("b"))
1
```